### DISCUSSION TITLE
cargo:warning=Failed to run protoc:

### DISCUSSION THREAD

**Discussion by ashwaq06** [nexus-xyz/nexus-zkvm#324] [Q&A]
<img width="1436" alt="image" src="https://github.com/user-attachments/assets/c047f3af-ec24-4a67-a7e5-19f807a92336" />
After executing the one-liner curl command, I’m encountering this issue. Could anyone please let me know if there’s anything missing in my setup?

**Comment by sjudson** [comment]
Please redirect to https://github.com/nexus-xyz/network-api
